tests/Aspire.Hosting.Testing.Tests/TestingBuilderTests.cs

```diff
@@ -2,7 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
I can't leave a comment on the empty file above, so I'm doing it here. Looks like the `Extensions.cs` file above is empty.